vmm/src/migration.rs

```diff
@@ -59,3 +62,25 @@ pub fn recv_vm_snapshot(source_url: &str) -> std::result::Result<Snapshot, Migra
         ))),
     }
 }
+
+pub fn vm_config_from_snapshot(
+    snapshot: &Snapshot,
+) -> std::result::Result<Arc<Mutex<VmConfig>>, MigratableError> {
+    if let Some(vm_section) = snapshot
+        .snapshot_data
+        .get(&format!("{}-section", VM_SNAPSHOT_ID))
+    {
+        let vm_snapshot: VmSnapshot =
+            serde_json::from_slice(&vm_section.snapshot).map_err(|e| {
+                MigratableError::Restore(anyhow!("Could not deserialize VM snapshot {}", e))
+            })?;
+
+        //        self.set_state(&ioapic_state);
```

Comment:
Strange comment here?

Reply:
Crufts...This needs to be removed indeed.